Scripts/Miscellaneous/Connect4Game/connect4game.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+from tkinter import *
+import pygame
+import sys
+import math
+
+BLACK = (33, 33, 33)
+GREY = (235, 235, 235)
+RED = (255,0,0)
+YELLOW = (255,255,0)
+
+ROW_COUNT = 6
+COLUMN_COUNT = 7
+
+player_names = []
+
+#adding players
+def submit():
+	player_names.append(player_one.get())
+	player_names.append(player_two.get())
+	root.destroy()
+
+#creating connect 4 board 6 X 7 common board size
+def create_board():
+	board = np.zeros((ROW_COUNT,COLUMN_COUNT))
+	return board
+
+def drop_piece(board, row, col, piece):
+	board[row][col] = piece
+
+def is_valid_location(board, col):
+	return board[ROW_COUNT-1][col] == 0
+
+def get_next_open_row(board, col):
+	for r in range(ROW_COUNT):
+		if board[r][col] == 0:
+			return r
+
+# def print_board(board):
```

Comment:
Remove this commented code

Reply:
The commented code has been removed from the source code.